### PR TITLE
Initiate project components

### PR DESCRIPTION
**Pull request by Mukumbuta**
### Features implemented:
- Set up React with the `create-react-app` command
- Set up linters
- Ste up React `BrowserRouter`.
- Set up `UUID` for dynamic generation of book IDs
- Added React Router and set two Routes and Links for the app's navigation: `Books` and `Categories`
- Made Books route as the default view
- Added functionality to display the list of books (empty at this point but it is ready for the data) with the Remove button (no functionality yet)
- Created a component called Book for displaying a single book (receive a title and an author as props)
- Reused the Book component in the component that displays a list of books
- Added a form component for adding a book (with inputs for a title and an author)
- Categories route displays a button "Check status" only.

### CODE REVIEW

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi @Mukumbuta 
I'm sure you put a lot of effort into this project but this PR is invalid because your README file is missing almost all of the important sections 😓 kindly copy everything from this [template](https://github.com/microverseinc/readme-template) 

Kindly fix the issue and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mukumbuta** [comment]
Hi @ha-manel
Please check through my README.md file in files changed tab. It contains all the important sections. I don't know what else is missing in there.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,182 @@
...
+# 📖 [CMS_Bookstore] <a name="about-project"></a>
+
+**[CMS_Bookstore]** is an MVP version of the Bookstore website that allows a user to:  Display a list of books; Add a book; and Remove a selected book. Built using React and Redux.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
```
Comment (Mukumbuta) [inline README.md:45]: Look, everything is here

**Review by Mukumbuta — COMMENTED** [review]
Everything is here

File: src/components/StatusButton.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const StatusButton = () => (
```
Comment (whiz25) [inline src/components/StatusButton.js:3]: - This component should be named "Categories". Please change that.

**Review by whiz25 — CHANGES_REQUESTED** [review]
Hi @Mukumbuta, 

Good job so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🎉 🎉 🎉 
- Codebase is organized 
- Readme looks professional 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/StatusButton.js (hunk 2 above)
Comment (Mukumbuta) [reply src/components/StatusButton.js:3]: Hi @whiz25
That is not the Categories component. I just created a re-usable StatusCheckButton component that I'm importing in the actual Categories component. So I don't know what you want me to do.

Comment (Mukumbuta) [reply src/components/StatusButton.js:3]: If you check through my files, you will find the actual Categories component

**Review by t-yanick — APPROVED** [review]
Hi @Mukumbuta, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**